.github/workflows/ts_declarations.yml

```
@@ -74,10 +74,35 @@ jobs:
         npm run validate-ts
         npm run validate-declarations
 
+  check-ts-regression-cases:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+    - name: Get sources
+      uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '15'
+
+    - name: Restore npm cache
+      uses: actions/cache@v2
+      with:
+        path: ./node_modules
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: ${{ runner.os }}-node-modules
+
+    - name: Run npm install
+      run: npm install --no-audit --no-fund
+
+    - name: Run TS tests
+      run: npm run test-typescript
```

```suggestion
    - name: Check sources compilation
      run: npm run validate-ts

    - name: Check test cases
      run: npm run test-typescript
```
And remove `npm run validate-ts` from the `'validate-declarations'` job